test/fx/test_z3_gradual_types.py

```diff
@@ -1463,6 +1463,45 @@ def forward(self, x: TensorType([20, 20])):
 
 
 class TestSingleOperation(unittest.TestCase):
+
+    def test_conv_wrong_example(self):
+        class BasicBlock(torch.nn.Module):
+            def __init__(self):
+                super(BasicBlock, self).__init__()
+                self.conv1 = torch.nn.Conv2d(in_channels=2, out_channels=2,
+                                             kernel_size=2, stride=2,
+                                             padding=2, groups=2, bias=False, dilation=2)
+
```

Comment:
It might make sense to add a torch.  A torch.nn.ReLU() after each Conv2 to make this more realistic.

Reply:
Added a relu after every conv